tools/accuracy/accuracy_checker.py

```diff
@@ -65,6 +65,21 @@ def parse_args():
                         help='specify input parameter dimension \
                                 with the following format --input_dim input_name:dim0,dim1,dim2...'
                         )
+    parser.add_argument('--target',
+                        type=str,
+                        default="gpu",
```

Comment:
```suggestion
                        default='gpu',
```
Just for consistency